vowpalwabbit/learner.h

```diff
@@ -63,7 +63,7 @@ struct learn_data
 };
 
 struct sensitivity_data
-{ using fn = float(*)(void* data, base_learner& base, void* ex);
+{ using fn = float(*)(void* data, base_learner& base, example& ex);
```

Comment:
Is there an inherent reason to assume that sensitivity only applies to example and not multi_ex?

Reply:
At least from my experience implementing RegCB and active e-greedy, I do not see a generic way to define sensitivity for multi_ex, and even if we wanted it to be algorithm-specific, it should probably return a vector instead of a single float, meaning that the `void*` approach still wouldn't be suitable. There might be a better way to approach things long term, but this seemed like a simpler fix for now.